src/documents/TextfieldStepperAPI.md

```diff
@@ -0,0 +1,340 @@
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dl>
+                    <dt>container</dt>
+                    <dd>
+                        A CSS-based selectors, single-element jQuery object, or DOM element that identifies the root DOM node where the Textfield Stepper should be placed.
+                    </dd>
+                </dl>
+                <dl>
+                    <dt>options</dt>
+                    <dd>
+                        An optional data structure that configures the Textfield Stepper component, as described below.
+                    </dd>
+                </dl>
+            </td>
+        </tr>
+        <tr>
+            <th>Returns</th>
+            <td>The Textfield Stepper component</td>
+        </tr>
+        <tr>
+            <th>Examples</th>
+            <td>
+<pre>
+<code>
+var textfieldStepper = fluid.textfieldStepper(".flc-textfieldStepper", {
+    strings: {
+        "label": "Textfield Stepper"
+    },
+    model: {
+        value: 1,
+        step: 0.1,
+        range: {
+            min: 1,
+            max: 2
+        }
+    },
+    scale: 1
+});
+</code>
+</pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Methods ##
+
+### increase ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textfieldStepper.increase();</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>increase</code> method increases the model value by the <code>model.step</code> amount.
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                By default this method does not take any arguments.
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### decrease ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textfieldStepper.decrease();</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>decrease</code> method decreases the model value by the <code>model.step</code> amount.
+            </td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                By default this method does not take any arguments.
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### addFocus ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Method</th>
+            <td>
+                <code>textfieldStepper.addFocus();</code>
+            </td>
+        </tr>
+        <tr>
+            <th>Description</th>
+            <td>
+                The <code>addFocus</code> method adds the focus style classes to the <code>focusContainer</code>.
```

Comment:
Didn't add a link, but added more information to indicate that `focusContainer` is a selector.